blackbox/docs/config/license.rst

```diff
@@ -0,0 +1,16 @@
+.. _conf-license:
+
+================
+License Settings
+================
+
+The section documents settings that only apply to the CrateDB Enterprise License.
+
+``expiration_date``: *expiration_date* (default: ``null``)
```

Comment:
`default` feels a bit strange - imho all fields should be required. So I'd remove these.

Reply:
agree - I'll remove the default